.github/workflows/adminBundleSize.yml

```diff
@@ -27,8 +27,6 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 18
-          path: '**/node_modules'
```

Comment:
Are you sure this can be removed?

Reply:
If it's replaced by the new custom install action: Totally sure. Context explained in the issue description. 

Ps from my phone I see only a line I might miss context...

Reply:
I've added the install: https://github.com/strapi/strapi/pull/16581/commits/e94f5eebcf84c8c75bb2ee7a2be631fa7732b3ab

But I'm not sure if the preact size action is a good way to do it. If interested we could create another issue about this.

Reply:
One step at a time :) If you have more tooling ideas I'm happy to help you in another PR, but in order to merge this one we need to keep the change small and manageable because quite a few developers depend on it :)

Reply:
Top !